lib/web_ui/test/ui/line_metrics_test.dart

```diff
@@ -128,6 +128,26 @@ Future<void> testMain() async {
     paragraph.layout(const ui.ParagraphConstraints(width: 400));
 
     expect(paragraph.numberOfLines, 1);
+    expect(paragraph.height, 10);
+
+    final ui.LineMetrics? metrics = paragraph.getLineMetricsAt(0);
+    expect(metrics, isNotNull);
+
+    // FlutterTest font's 'X' character is a square, so it's the font size (10.0) * 4 characters.
+    expect(metrics!.width, 40.0);
+  });
+
+  test('uses flutter test font by default when debugEmulateFlutterTesterEnvironment is enabled', () {
```

Comment:
Might be a good idea to add a test where you set `debugEmulateFlutterTesterEnvironment` to false and make sure the effective font families are the same as the provided ones and not the test ones.

Reply:
Good call.